src/main/java/bio/terra/service/snapshotbuilder/utils/AggregateBQQueryResultsUtils.java

```diff
@@ -18,4 +18,10 @@ public static List<SnapshotBuilderConcept> aggregateConceptResults(TableResult r
                     .count(1))
         .toList();
   }
+
+  public static List<Integer> rollupCountsMapper(TableResult result) {
+    return StreamSupport.stream(result.iterateAll().spliterator(), false)
+        .map(row -> (Integer) row.get(0).getValue())
```

Comment:
Is it possible for `getValue()` to return `null`? If not I would instead cast to `int` here, to make it clear that it can't be `null`.
```suggestion
        .map(row -> (int) row.get(0).getValue())
```